src/Common/Scheduler/Workload/WorkloadEntityDiskStorage.cpp

```diff
@@ -117,61 +120,81 @@ ASTPtr WorkloadEntityDiskStorage::tryLoadEntity(WorkloadEntityType entity_type,
 }
 
 
-void WorkloadEntityDiskStorage::loadEntities()
+void WorkloadEntityDiskStorage::loadEntities(const Poco::Util::AbstractConfiguration & config)
 {
     if (!entities_loaded)
+    {
+        // Load config entities first
+        config_storage->loadEntities(config);
```

Comment:
Make sure config entities are updated even after initial load of keeper entities. Fix this code